addok/ds.py

```diff
@@ -56,7 +56,9 @@ def store_documents(docs):
     for doc in docs:
         if not doc:
             continue
-        key = keys.document_key(doc['id'])
+        if '_hashid' not in doc:
```

Comment:
I don't know if we want to use the `_` (underscore) prefix here and if we want the end-user to know about it. Related to the GET possibility.